dotnetv3/Keyspaces/Tests/CassandraWrapperTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace KeyspacesScenario.Tests
+{
+    /// <summary>
+    /// Tests for the CassandraWrapper class.
+    /// </summary>
+    public class CassandraWrapperTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IAmazonKeyspaces _client;
+        private readonly CassandraWrapper _wrapper;
+        private readonly string _keyspaceName;
+        private readonly string _tableName;
+        private readonly string _movieFileName;
+        private readonly int _moviesToInsert = 5;
+        private static string _movieToUpdate;
+        private static int _movieToUpdateYear;
+
+        /// <summary>
+        /// Constructor for the test class.
+        /// </summary>
+        public CassandraWrapperTests()
+        {
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from .json file.
+                .AddJsonFile("testsettings.local.json",
+                    true) // Optionally load local settings.
+                .Build();
+
+            _client = new AmazonKeyspacesClient();
+            _wrapper = new CassandraWrapper();
+
+            _keyspaceName = _configuration["KeyspaceName"];
+            _tableName = _configuration["TableName"];
+
+            _movieFileName = _configuration["MovieFile"];
+
+        }
+
+        /// <summary>
+        /// Tests the ImportMoviesFromJson method by importing 5 movies. The
+        /// number of movies in the move list should be 5 after the call.
```

Comment:
remove the word 'move'

Reply:
This should have said "movies in the movie list" but I missed the i in the second movie.